modules/kubernetes/aks-core/modules.tf

```diff
@@ -310,3 +310,19 @@ module "azad_kube_proxy" {
     scope         = var.azad_kube_proxy_config.k8dash_config.scope
   }
 }
+
+# platform
+module "platform" {
+  for_each = {
+    for s in ["platform"] :
+    s => s
+    if var.platform_enabled
+  }
+
+  source = "../../kubernetes/platform"
+
+  platform_config = var.platform_config
```

Comment:
Config should be used in the aks_core module but we should avoid using these types of objects in the root module.